edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/KubernetesModule.cs

```diff
@@ -177,8 +177,9 @@ protected override void Load(ContainerBuilder builder)
                     c =>
                     {
                         var loggerFactory = c.Resolve<ILoggerFactory>();
-                        var kubernetesCommandFactory = new KubernetesCommandFactory();
-                        ICommandFactory factory = new LoggingCommandFactory(kubernetesCommandFactory, loggerFactory);
+                        ICommandFactory factory = new KubernetesCommandFactory();
+                        factory = new MetricsCommandFactory(factory);
```

Comment:
nit: suggest to have extension method to create MetricsCommandFactory / LoggingCommandFactory to make it more readable.

Reply:
is this hard to read? how would an extension function clean this up?